variables.robot

```diff
@@ -337,7 +345,7 @@ ${OS_UBUNTU}=               ubuntu
 ...                         sbo_name=USB    name=SanDisk Ultra USB 3.0
 &{USB13}=                   vendor=Artificial    volume=1GB    type=USB_Storage
 ...                         protocol=3.0    interface=USB    count=1
-...                         sbo_name=USB    name=PiKVM Composite KVM Device
```

Comment:
Why do we drop `Device` ?

Reply:
Limitation on AMI https://github.com/Dasharo/open-source-firmware-validation/commit/6685ced3689cfb524fb687d14855f4e557e12c11 We couldn't add whole name there